### DISCUSSION TITLE
what can i upload to your site

### DISCUSSION THREAD

**Discussion by Glenniemay** [community/community#126525] [Programming Help] [answered]
### Select Topic Area

General

### Body

hi, this site is very technical, espesh for a 70 year old lady. I just came across it from google whilst looking for how to convert a certain type of audio file - aax. Then joined back up with Github, reset password and other stuff. Kept trying to get that .aax converter to work because the instructions didn't seem to be correct. Also dloaded BookLib and GitHub Desktop and installed it. Just need to know what I can do for you, or upload to you. I have no idea how to create a project or repository - thought that was a thing people used internally for constipation??!! anyway, I'm prob not even on the correct branch of your huge site, but maybe someone will read it. To get the torrent software "qBittorrent", do I just type it in one of your search bars?? sorry to bother you, just feel a bit lost.

**Comment by dhruv8433** [comment, marked as answer]
@Glenniemay 

`qBittorrent` is a free and open-source torrent client. You don't need to search for it on GitHub unless you're interested in the source code. You can download it directly from its official website:

Download qBittorrent:
1. Go to the official qBittorrent website.
2. Choose the version appropriate for your operating system (Windows, Mac, or Linux) and download it.
3. Follow the installation instructions on the website.

**Reply by dhruv8433** [reply]
@Glenniemay 

Mark this as answer

**Comment by Glenniemay** [comment]
Thanks very much Soni.

Kind regards

Glennie

**Comment by dhruv8433** [comment]
@Glenniemay 

can you please select my answer and set as 'answered' and close this QA discussion

**Comment by Glenniemay** [comment]
Of course I will do that. Just give me a bit of time as I’m preparing dinner right now.

Thanks again Soni

Glennie

**Comment by Glenniemay** [comment]
As requested by Soni Dhruv, I will try to follow his instructions and find the "set as answered" option.
Thanks again.
Glenniemay

**Comment by Glenniemay** [comment]
Hope I did this correctly

**Comment by Glenniemay** [comment]
Hello again, I logged in and hopefully marked the discussion as answered correctly, as you requested.

Got a green tick afterwards anyway, so it all appears as taken care of now?

Let me know if you need me to take more action on this, because as you know, I’m a complete newbie.

Cheers

Glenniemay

**Comment by dhruv8433** [comment]
@Glenniemay 

Select my first comment as a answer
this one

![image](https://github.com/community/community/assets/114583978/4351425f-9dcd-43f0-8819-3129c520376f)


Not email replay

**Reply by Glenniemay** [reply]
marked as answer.

**Comment by Glenniemay** [comment]
Hi, I just did that again, marked as answer -  twice, so hopefully it worked out.

Thanks

G

**Comment by dhruv8433** [comment]
@Glenniemay 

still it is unanswerd 
![image](https://github.com/community/community/assets/114583978/6744c8ce-30c0-46e6-895b-edb024dc607a)

you have to mark first comment as answer
i past link of that please click on this link and mark first comment as answer
https://github.com/orgs/community/discussions/126525#discussioncomment-9602477

you got green right in right side when answer is selected
![image](https://github.com/community/community/assets/114583978/9eca0b85-15f9-4d3a-a77b-1d6df79c519b)

**Comment by Glenniemay** [comment]
That’s the best I can do Soni. The page where there’s a box saying “mark as answer” is greyed out.

I logged out and back in and tried again. Clicked what was available to answer.
cheers
